### PR TITLE
Cosmetic frontend fixes

### PR DESCRIPTION
**Pull request by bfokss**
Added:

- [Listing users]
- [Adding users]
- [Assigning users to cards]
- [Listing cards]
- [Adding cards]
- [Simple navigation on Home page]

Fixed:

- [ Fetching all trainings and limiting it on frontend ]
- [ Couple small things on views ]

?

I'm not proud of the     
  
        user_id = request.form['user_id']
        user_id = user_id.replace(" ","")
        user_id = user_id.split('-')
        user_id = user_id[0]

workaround parts. If you have any other better ideas, please let me know!

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -36,13 +36,113 @@ def user_index():
...
+
+        usersData = pd.DataFrame(users)
+        cardsData = pd.DataFrame(cards)
+
+        return render_template('main/assign_user.html', usersData=usersData, cardsData=cardsData)
+    
+    if request.method == 'POST':
+        
+
+        user_id = request.form['user_id']
```
Comment (wdebsqi) [inline app.py:97]: FYI you can easily clean the text in Python using regex. Here's an example of how you could be cleaning text from non-alphanumerical characters.

```
import re
uncleaned = "tes t-i-n   g 1@2*3!!!"
clean = re.sub(r"[^a-z^\d]", "", uncleaned)
clean
```

Output: `"testing123"`

`^a-z` stands for non-alphabetical and `^\d` for non-numerical.

File: app.py [hunk 2]
```diff
@@ -36,13 +36,113 @@ def user_index():
...
+            Card.user_id
+        ).select_from(Card).all()
+
+        usersData = pd.DataFrame(users)
+        cardsData = pd.DataFrame(cards)
+
+        return render_template('main/assign_user.html', usersData=usersData, cardsData=cardsData)
+    
+    if request.method == 'POST':
+
```
Comment (wdebsqi) [inline app.py:95]: Redundant new lines.

**Review by wdebsqi — APPROVED** [review]
Looks really good :)
You can implement changes I mentioned in app.py file. But it's not a must-have. You choice. Merge when you're ready.

File: app.py (hunk 1 above)
Comment (wdebsqi) [reply app.py:97]: Btw. here's a nice website for testing regular expressions: https://regex101.com/

Comment (bfokss) [reply app.py:97]: Thanks for pro tip :) Used it and fixed the problem

File: app.py (hunk 2 above)
Comment (bfokss) [reply app.py:95]: Didn't fix this one. Please fix it while working on backend update.

**Pull request merged**